experiments/task.py

```diff
@@ -117,13 +112,13 @@ def setup(self):
             rwd = block["rwd"]
 
             for i in range(n):
-                c1 = c2 =  np.searchsorted(P_cue, random.uniform(0,1))
+                c1 = c2 =  np.searchsorted(P_cue, np.random.uniform(0,1))
```

Comment:
I see. I did a quick n' dirty check, and on my machine, with Python 3.5.1, `random` seem actually slower than `np.random` for a single scalar:

```python
import time
import random
import numpy as np

N = 10000000

start = time.time()
for _ in range(N):
    np.random.uniform(0,1)
print(time.time() - start)

start = time.time()
for _ in range(N):
    random.uniform(0,1)
print(time.time() - start)
```
returns:
```
3.8670051097869873
5.967450857162476
```